content/applications/inventory_and_mrp/manufacturing/management/scrap_manufacturing.rst

```diff
@@ -0,0 +1,110 @@
+Finally, click :guilabel:`Done` to scrap the component.
+
+.. image:: scrap_manufacturing/scrap-pop-up.png
+   :align: center
+   :alt: The Scrap pop-up window.
+
+.. tip::
+   Before clicking :guilabel:`Mark As Done` on a manufacturing order, only the components of the
+   finished product can be scrapped, **not** the finished product itself. This is because Odoo
+   recognizes that the finished product cannot be scrapped before it has been manufactured.
+
+After scrapping a component, continue the manufacturing process using the required quantity of the
+component that was scrapped. The on-hand stock count for the component that was scrapped updates to
+reflect both the scrapped quantity and the quantity consumed during manufacturing.
+
+.. example::
+   If the manufacturing of Product X requires 10 units of Component Y, and five units of Component Y
+   were scrapped during the manufacturing process, the total quantity of Component Y consumed will be
+   15: 10 units used to manufacture Product X + five units scrapped.
+
+Scrap components from tablet view
+=================================
+
+Components can also be scrapped from the manufacturing tablet view. To do so, select the
+:guilabel:`Work Orders` tab on a manufacturing order, then click the :guilabel:`tablet view` icon
+for a work order.
+
+.. image:: scrap_manufacturing/tablet-view-icon.png
+   :align: center
+   :alt: The tablet view icon for a work order.
+
+With tablet view open, click the :guilabel:`☰` (menu) button at the top left of the screen, then
+select the :guilabel:`Scrap` button on the :guilabel:`Menu` pop-up window. The :guilabel:`Scrap`
+pop-up window then appears.
+
+.. image:: scrap_manufacturing/tablet-scrap-button.png
+   :align: center
+   :alt: The scrap button on the menu pop-up window of the manufacturing tablet view.
+
+Finally, select a component from the :guilabel:`Product` drop-down menu and enter the quantity being
+scrapped in the :guilabel:`Quantity` field. Click :guilabel:`Done` to scrap the component.
+
+Scrap finished products
+=======================
+
+Odoo also allows for finished products to be scrapped from a manufacturing order once the order is
+completed. After clicking :guilabel:`Mark as Done`, click the :guilabel:`Scrap` button to make the
+:guilabel:`Scrap` pop-up window appear.
+
+Since the components have been consumed to create the finished product, they will no longer appear
+in the :guilabel:`Product` drop-down menu. Instead, the finished product will be available as an
+option. Select the finished product and enter the quantity to be scrapped in the
+:guilabel:`Quantity` field. Click :guilabel:`Done` to scrap the finished product.
+
+The on-hand stock count for the product that was scrapped will update to reflect both the scrapped
+quantity and the quantity produced during manufacturing.
+
+.. example::
+   If five units of Product Z were manufactured, but two units were scrapped after manufacturing was
```

Comment:
Again, this may be a stylistic choice to capitalize "Product", leave this if you feel strongly about it :)
```suggestion
   If five units of product Z were manufactured, but two units were scrapped after manufacturing was
```